tools/webhook-apicoverage/view/coverage_display.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package view
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/knative/test-infra/tools/webhook-apicoverage/coveragecalculator"
+)
+
+// GetCoverageValuesDisplay builds the string display for coveragecalculator.CoverageValues
+func GetCoverageValuesDisplay(coverageValues *coveragecalculator.CoverageValues) string {
+	var buffer strings.Builder
+	buffer.WriteString("\nCoverage Values:\n")
+	buffer.WriteString(fmt.Sprintf("Total Fields : %d\n", coverageValues.TotalFields))
+	buffer.WriteString(fmt.Sprintf("Covered Feilds : %d\n", coverageValues.CoveredFields))
+	buffer.WriteString(fmt.Sprintf("IgnoredFields : %d\n", coverageValues.IgnoredFields))
+
+	percentCoverage := 0.0
+	totalFields := coverageValues.TotalFields
+	// To handle cases where a resource has covered ignored fields.
```

Comment:
Uh... Shouldn't `totalFields` always be `coverageValues.TotalFields - coverageValues.IgnoredFields`?

Reply:
Actually there were cases where some fields were added to ignored that are in fact covered, as a result TotalFields - IgnoredFields ended up being less than covered fields, giving coverage percentage to be greater than 100.

Reply:
Sounds like an error to me, and thus shouldn't be handled this way.

Reply:
Thinking more, I think it would be better to always have it `coverageVaules.TotalFields - coverageValues.IgnoredFields` and have the repo owners fix the percentage by appropriately updating their ignoredFields.yaml file. So have changed it.

Reply:
> Sounds like an error to me, and thus shouldn't be handled this way.

Yes fixed it.